src/lib/select/_select-theme.scss

```diff
@@ -11,12 +11,12 @@
     color: md-color($foreground, hint-text);
     border-bottom: 1px solid md-color($foreground, divider);
 
-    md-select:focus & {
+    md-select:focus:not([aria-disabled='true']) & {
```

Comment:
Can we add a class for this instead? Using the `aria-disabled` makes the selectors somewhat noisy.